mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -215,6 +215,10 @@ static cl::opt<std::string> tensorDataType("t", cl::desc("Data type for convolut
                                            cl::value_desc("Data type for convolution"),
                                            cl::init("f32"));
 
+static cl::opt<std::string>
+    randomData("rand", cl::desc("To use random data for host validation"),
+               cl::value_desc("To use random data for host validation"),
```

Comment:
please help specify what the argument is for.

Reply:
I've changed the description to specify the argument is used for giving a seed of random number generator.